src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
+public class UserService {
     public int getUserScore(String[] records, String email) {
-        return 0;
+        String getRecods = "";
+        for (String record : records) {
+            if (email.equals(record.split(":")[0])) {
```

Comment:
let's firstly save the result of operation `record.split(":")[0])` 
and then we can use this data for checking and getting the result

Reply:
how about this comment?

Reply:
add question to slack